lib/transaction.c

```diff
@@ -1415,8 +1467,23 @@ static int rpmtsSetup(rpmts ts, rpmprobFilterFlags ignoreSet)
     return 0;
 }
 
+/* Enable / disable optimizations for solid state disks */
+static void setSSD(int enable)
+{
+    if (enable) {
+	rpmlog(RPMLOG_DEBUG, "optimizing for non-rotational disks\n");
+	rpmPushMacro(NULL, "_minimize_io", NULL, "1", 0);
```

Comment:
A quick look-up in the rpm repo reveals there's no `_minimize_io`. Are you sure this is not a mistake? :)

Reply:
Whoops. Indeed, that should be `_minimize_writes` instead, thanks for spotting. Also goes to show how tested (or not) this is, I guess :smile:

Reply:
Macro name fixed in the push. Other caveats still apply...